src/gui/options.ui

```diff
@@ -719,6 +719,64 @@
                  </layout>
                 </widget>
                </item>
+               <item row="2" column="0">
+                <widget class="QGroupBox" name="deleteTorrentBox">
+                 <property name="toolTip">
+                  <string>Should the .torrent file be deleted after adding it</string>
+                 </property>
+                 <property name="title">
+                  <string>Delete .torrent files afterwards </string>
+                 </property>
+                 <property name="checkable">
+                  <bool>true</bool>
+                 </property>
+                 <property name="checked">
+                  <bool>false</bool>
+                 </property>
+                 <layout class="QVBoxLayout" name="verticalLayout_17">
+                  <item>
+                   <widget class="QCheckBox" name="deleteCancelledTorrentBox">
+                    <property name="toolTip">
+                     <string>Delete also files for torrent whose addition was cancelled</string>
```

Comment:
Thenk you. Can you also suggest how to simplify "files for torrent whose addition", please?
"Also delete .torrent files whose addition was cancelled"?